resources/views/site/digitalPublicationDetail.blade.php

```diff
@@ -157,17 +199,29 @@
                     @break
 
                     @case('entries')
+
+                        {{-- Title Component --}}
+                        @if (!$topLevelArticle->hide_title)
+                            @component('components.molecules._m-title-bar', [
+                                'variation' => 'm-title-bar--compact m-title-bar--light',
+                            ])
+                            @slot('links', ((count($topLevelArticle->children) > 4 && count($topLevelArticle->children) < 8) || (count($topLevelArticle->children) > 8)) ? $topLevelArticle->present()->getBrowseMoreLink($showAll) : [])
```

Comment:
This is hard to read. I think checking if the number of children is not equal to 0, 4, or 8 might be easier to understand.